source/_lovelace/picture-elements.markdown

```diff
@@ -74,6 +74,11 @@ tap_action:
   description: more-info, toggle, navigate, call-service
   type: string
   default: more-info
+hold_action:
+  required: false
+  description: Action to perform when clicked-and-held. more-info, toggle, navigate, call-service
```

Comment:
Action to perform when clicked-and-held (e.g., `more-info`, `toggle`, `navigate`, `call-service`)

Reply:
Done. thx :)